AccessServices/Infrastructure/ProductService.cs

```diff
@@ -0,0 +1,76 @@
+﻿using AccessServices.Interfaces;
+using AutoMapper;
+using DataAccess.Interfaces;
+using DataAccess.Infrastructure.EfCore;
+using DataAccess.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using AccessServices.Mapper;
+using AccessServices.DTOs;
+using AccessServices.Helpers;
+
+namespace AccessServices.Infrastructure
+{
+    public class ProductService
+    {
+
+        readonly IMapper mapper;
+        public ProductService()
+        {
+            var mappingConfig = new MapperConfiguration(mc =>
+            {
+                mc.AddProfile(new MappingProfile());
+            });
+            mapper = mappingConfig.CreateMapper();
+        }
+
+        public IEnumerable<ProductDTO> GetProducts()
+        {
+            using (var dbAccesser = new DbAccesserEF<Product>(BindingsHendler.GetContext()))
+            {
+                var dbItems = dbAccesser.GetItems();
+                return mapper.Map<IEnumerable<Product>, IEnumerable<ProductDTO>>(dbItems);
+            }
+        }
+
+        public void AddProduct(ProductDTO product)
+        {
+            using (var dbAccesser = new DbAccesserEF<Product>(BindingsHendler.GetContext()))
+            {
+                var dbAccesserCategory = new DbAccesserEF<Category>(BindingsHendler.GetContext());
+                var categ = dbAccesserCategory.GetItem(c => c.Name == product.CategoryName);
+                var newProduct = mapper.Map<ProductDTO, Product>(product);
+                newProduct.CategoryId = categ.Id;
+                dbAccesser.AddItem(newProduct);
+            }
+        }
+
+        public void EditProduct(ProductDTO product)
+        {
+            using (var dbAccesser = new DbAccesserEF<Product>(BindingsHendler.GetContext()))
```

Comment:
An approach of using 'new' keyword is not applicable if you are trying to set up DI approach.
Let's sync about it separately.